### DISCUSSION TITLE
Is there support for C# dev kit?

### DISCUSSION THREAD

**Discussion by ChimpSpin** [zed-industries/zed#22234] [Feature Requests]
I see some talk about it here, but the thread has died. https://github.com/zed-industries/zed/pull/6908 Has work stopped?

Omnisharp times out during initialization on large projects for me.

**Comment by 0x2CA** [comment]
Roslyn Lsp is a future option

But as far as I know, the C# dev kit is a closed-source extension, exclusively provided by Microsoft for use with VSCode.

Because Microsoft has once again betrayed open source, any other third-party editor also just uses C# Lsp

**Reply by ChimpSpin** [reply]
You are right. Sorry, I meant Roslyn Lsp.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
